### ISSUE TITLE
Add group options to apps

### ISSUE THREAD

**Issue by slontis** [openssl/openssl#9936]
PR #9920 introduced some option grouping for displaying 'groups' of command line arguments.
openssl apps that have many arguments would benefit from this functionality..

**Comment by richsalz** [comment]
I'll work on this.  This script shows how many options are in each command:
```
for I in `./apps/openssl list -1 -commands` ; do
   echo $(./apps/openssl $I --help 2>&1 | wc -l) $I
done | sort -nr
```
I suggest any command with fewer than 20 lines doesn't need this.  Thoughts?

**Comment by paulidale** [comment]
I suggest applying groupings consistently irrespective of the number of lines.

**Comment by richsalz** [comment]
I dunno, this looks a little silly to me.  But whatever the project wants:
```
; ./apps/openssl errstr --help
Usage: errstr [options] errnum...
  errnum  Error number
General options:
 -help  Display this summary
; 
```

**Comment by levitte** [comment]
Er, what part of this has anything to do with the new core or FIPS????

**Comment by mattcaswell** [comment]
This was fixed by #9953
